ssh/generate.go

```diff
@@ -4,62 +4,123 @@
 package ssh
```

Comment:
this generation work done in `generate.go` should really come from the new `pki/ssh` package. I'd prefer to retire this whole `ssh` package.